google/resource_compute_router_bgp_peer.go

```diff
@@ -198,6 +199,12 @@ func resourceComputeRouterBgpPeerCreate(d *schema.ResourceData, meta interface{}
 	} else if v, ok := d.GetOkExists("interface"); !isEmptyValue(reflect.ValueOf(interfaceNameProp)) && (ok || !reflect.DeepEqual(v, interfaceNameProp)) {
 		obj["interfaceName"] = interfaceNameProp
 	}
+	ipAddressProp, err := expandComputeRouterBgpPeerIpAddress(d.Get("ip_address"), d, config)
+	if err != nil {
+		return err
+	} else if v, ok := d.GetOkExists("ip_address"); !isEmptyValue(reflect.ValueOf(ipAddressProp)) && (ok || !reflect.DeepEqual(v, ipAddressProp)) {
```

Comment:
Just wondering how does this check that provided IP address is `IPv4` not `IPv6` ?